compiler/src/latex_printer.ml

```diff
@@ -235,6 +235,18 @@ let pp_args fmt (sty, xs) =
     pp_sto_ty sty
     (pp_list " " pp_var) xs
 
+let pp_decl fmt (x: vardecl L.located) =
+  match L.unloc x with
+  | NotInitVarDecl v -> F.fprintf fmt "%s" (L.unloc v)
+  | InitVarDecl (v, e) -> F.fprintf fmt "%s = %a" (L.unloc v) pp_expr e
+
+let pp_decls fmt (sty, vd) =
+  F.fprintf
+    fmt
+    "%a %a"
+    pp_sto_ty sty
+    (pp_list " " pp_decl) vd
```

Comment:
Is it OK to have a blank separator when there is an initializing expression?

Reply:
To be honest, I have no opinion about this matter. Would you prefer it without the blank separator ?

Reply:
Yes it is ;-)

Reply:
I was not asking about opinion but about correctness. It is correct: I’ve thus enriched the example file.

Reply:
Sorry, I didn't understand your question ^^'